$NetBSD: patch-ab,v 1.1 2008/01/27 12:35:17 rillig Exp $

The construction #if FOO #elif FOO #endif is almost always wrong.

--- xiohelp.c.orig	2007-02-05 22:15:57.000000000 +0100
+++ xiohelp.c	2008-01-27 13:29:19.000000000 +0100
@@ -25,7 +25,7 @@ static const char *optiontypenames[] = {
 	"OFF_T",	"OFF64_T",
 #if HAVE_STRUCT_IP_MREQN
 					"STRUCT-IP_MREQN",
-#elif HAVE_STRUCT_IP_MREQN
+#elif HAVE_STRUCT_IP_MREQ
 					"STRUCT-IP_MREQ",
 #endif
 							"IP4NAME",
